docs/first-model/run-model.rst

```diff
@@ -1,11 +1,25 @@
 Run model
 =========
 
-.. warning::
+Tandem simulation can be run with a syntax of :code:`$ tandem parameter_file.toml {tandem options} --petsc {PETSc options}`.
```

Comment:
Tandem simulation**s** can be run with a syntax of

Reply:
What about:
Tandem is invoked with the following syntax